src/main/java/org/apache/commons/imaging/formats/tiff/fieldtypes/FieldTypeRational.java

```diff
@@ -31,11 +31,19 @@ public FieldTypeRational(final int type, final String name) {
     @Override
     public Object getValue(final TiffField entry) {
         final byte[] bytes = entry.getByteArrayValue();
+        boolean unsignedType = true;
+        if(entry.getFieldType()==RATIONAL){
+            unsignedType = true;
+        }else if(entry.getFieldType()==SRATIONAL){
+            unsignedType = false;
+        }
```

Comment:
What about:

```suggestion
        boolean unsignedType = entry.getFieldType() != SRATIONAL;
```

Reply:
That will work.  There are only the two conditions.

Reply:
done